front/app/api/me/useAuthUser.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import meKeys from './keys';
+import { MeKeys } from './types';
+import { IUser } from 'api/users/types';
+
+export const fetchMe = () =>
+  fetcher<IUser>({
+    path: `/users/me`,
+    action: 'get',
+  });
+
+const useAuthUser = () => {
+  return useQuery<IUser, CLErrors, IUser, MeKeys>({
+    queryKey: meKeys.all(),
+    queryFn: () => fetchMe(),
+    retry: false,
```

Comment:
Just not quite sure how it works - if we set the queryClient to have default `retry: false` --> would that apply then to all instances of useQuery? :thinking:

Reply:
Yes, this is no longer needed since we made the global change. I'll remove it.